CHANGELOG.md

```diff
@@ -445,3 +445,62 @@ All notable changes to this project will be documented in this file. The format
 ### Fixed
  - Fix: Certificate, from hostrule CRD, can not be assigned to L7 VirtualService when AKO is deployed in Dedicated mode.
  - Fix: Virtualservices are getting deleted when there is an issue with an access to the kube_api server which results in setting up `deleteConfig` flag to true.
+
+## AKO-1.12.1
+
+### Added
+- AKO now claims support for Kubernetes 1.29, OCP version 4.14.
+- AKO supports LoadBalancerClass for LoadBalancer type services.
```

Comment:
link to each feature page will be good so that user will be redirected to it for more details about how to use and what it is .

Reply:
@akshayhavile I have just copied the content for 1.12.x from their respective branches so that we just have it updated in master and 1.13.1. Those are already updated in the official broadcom docs at the time of 1.12.x release.

Reply:
@DixitAakash My bad. I wanted to comment on 1.13.1.  It will be good to have links pointing to feature docs.

Reply:
@akshayhavile I have added the required links.